app/code/Magento/Customer/Model/CustomerRegistry.php

```diff
@@ -148,9 +164,9 @@ public function retrieveSecureData($customerId)
         if (isset($this->customerSecureRegistryById[$customerId])) {
             return $this->customerSecureRegistryById[$customerId];
         }
-        /** @var Customer $customer */
+        /** @var CustomerResourceFactory $customer */
```

Comment:
It doesn't looks like correct fix, as phpdoc block shows it should return Customer model. Could you double check it?